.env.example

```
@@ -12,4 +12,12 @@ FORCE_ISOCODE=
 # Get your API key at: https://sdk-doc-liquid.breez.technology/guide/getting_started.html#api-key
-BREEZ_API_KEY=
\ No newline at end of file
+BREEZ_API_KEY=
+
+# --- Feature Flags ---
+# Enable the passkey flow for Cube creation (WebAuthn + PRF extension).
+# Requires macOS code signing with associated-domains entitlement + an AASA
+# file hosted at coincube.io/.well-known/apple-app-site-association.
+# Default: off. Set to 1 / true / yes to enable the "Use Passkey" toggle
+# in the Create Cube form.
+COINCUBE_ENABLE_PASSKEY=0
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)